### PR TITLE
Changes for ImageMagick 7 (rmdraw.c)

### PR DESCRIPTION
**Pull request by dlemstra**

### CODE REVIEW

File: ext/RMagick/rmdraw.c [hunk 1]
```diff
@@ -1140,12 +1183,23 @@ Draw_draw(VALUE self, VALUE image_arg)
     // Point the DrawInfo structure at the current set of primitives.
     magick_clone_string(&(draw->info->primitive), StringValuePtr(draw->primitives));
 
+#if defined(IMAGEMAGICK_7)
+    exception=AcquireExceptionInfo();
```
Comment (Watson1978) [inline ext/RMagick/rmdraw.c]: More beautiful with space around `=` 😊

File: ext/RMagick/rmdraw.c [hunk 2]
```diff
@@ -963,9 +978,25 @@ VALUE Draw_annotate(
     }
 
     // Translate & store in Draw structure
+#if defined(IMAGEMAGICK_7)
+    exception=AcquireExceptionInfo();
```
Comment (Watson1978) [inline ext/RMagick/rmdraw.c]: More beautiful with space around = 😊

File: ext/RMagick/rmdraw.c [hunk 3]
```diff
@@ -1731,25 +1788,53 @@ get_type_metrics(
     }
 
     Data_Get_Struct(self, Draw, draw);
+#if defined(IMAGEMAGICK_7)
+    exception=AcquireExceptionInfo();
```
Comment (Watson1978) [inline ext/RMagick/rmdraw.c]: More beautiful with space around = 😊

**Comment by dlemstra** [comment]
Thanks for the review! The spaces have been added.

**Pull request merged**